exonum-java-binding/core/rust/src/utils/services.rs

```diff
@@ -14,36 +14,46 @@
 
 use std::{collections::HashSet, fs::File, io::Read, path::Path};
 
+use std::collections::HashMap;
 use toml;
 
-pub const PATH_TO_SERVICES_TO_ENABLE: &str = "ejb_app_services.toml";
+pub const PATH_TO_SERVICES_DEFINITION: &str = "ejb_app_services.toml";
 pub const CONFIGURATION_SERVICE: &str = "configuration";
-pub const BTC_ANCHORING_SERVICE: &str = "btc-anchoring";
-pub const TIME_SERVICE: &str = "time";
-pub const EJB_SERVICE: &str = "ejb-service";
+pub const BTC_ANCHORING_SERVICE: &str = "btc_anchoring";
```

Comment:
Why this change, requiring all docs updates?
I think the former are just as good (also, '_' is not used in Java artifact names, our Rust similarly do not use it).

time is good too.

Reply:
Because of testing of `node_builder::prepare_service_factories()`. The only way to check ServiceFactory is presented in the resulting vector is to iterate over the vector and compare for `ServiceFactory.service_name()`. And the names of system services (in their `ServiceFactory` implementations) are defined as "btc_anchoring", "configuration" and "exonum_time". So I just synchronized our constants with them in order to not support two sets of constants.

Reply:
I see, but isn't it easier to add test-only constants than to have a configuration file dependent on the values from these services (and migrate all the docs, existent configuration files, putting that into changelog and the migration guide)?

Reply:
I'm in doubt these changes in constants are going to bring so many changes into the real picture, but I agree that it's better to add test-only constants.